builder/vsphere/clone/builder.go

```diff
@@ -135,6 +135,10 @@ func (b *Builder) Run(ctx context.Context, ui packersdk.Ui, hook packersdk.Hook)
 		&common.StepRemoveCDRom{
 			Config: &b.config.RemoveCDRomConfig,
 		},
+		&common.StepKeepCDRom{
```

Comment:
nitpick: like `CD`, `ROM` is an acronym too. so maybe, all these names should use `CDROM`.